config/routes.rb

```diff
@@ -10,5 +10,10 @@
 
   get 'dashboard', to: 'dashboard#index'
 
-  get '/movies', to: 'movies#index'
+  get '/discover', to: 'discover#index'
+  # if we want to next discover under movies we could use the non-restul route below...
+  # get '/discover', to: 'movies#discover'
```

Comment:
I think its okay to not be restful here if you want to move this to the movies controller I'm down